src/sentry/search/eap/resolver.py

```diff
@@ -36,10 +36,12 @@
 from sentry.search.eap import constants
 from sentry.search.eap.columns import (
     ColumnDefinitions,
+    CustomFunction,
     ResolvedColumn,
     ResolvedFunction,
     VirtualColumnDefinition,
 )
+from sentry.search.eap.span_columns import CUSTOM_FUNCTIONS
```

Comment:
You should pass this as part of the `ColumnDefinitions` as this file should remain independent of the dataset.

Reply:
fixed